tests/tt_metal/tt_metal/perf_microbenchmark/dispatch/test_pgm_dispatch.cpp

```diff
@@ -222,25 +222,24 @@ int main(int argc, char **argv) {
         initialize_program(program[1], fast_kernel_cycles_g);
 
         // Cache stuff
+        EnqueueProgram(cq, program[0], false);
+        EnqueueProgram(cq, program[1], false);
+        uint32_t tid = BeginTraceCapture(device, cq.id());
```

Comment:
We should still be supporting variants of this test that use host dispatch right? Should probably add a runtime option that allows us to toggle running this test in trace vs host dispatch mode.

Reply:
yes, both are important